playbooks/code-review.md

```diff
@@ -2,3 +2,45 @@
 title: Code reviews
 description: How our code review process works 
 ---
+
+Reviewing other people's code is a responsibility of every developer. Instead of viewing it as a hassle, it should be looked upon as a necessary process in order to keep the code clean and consistent.
+It is also an opportunity to learn something new, improve your mentoring skills and help someone else improve their skillset.
+
+Rules on..
+
+## How a Code Review process should look like
+
+..are written below.
+
+### Terms:
+ - PR - Pull Request.
+ - PR Creator - a person who initiated the PR.
+ - Reviewer - a person assigned to do a PR code review.
+
+### Rules:
+1. Every PR has to be reviewed, with the exception of fix/hotfix PR-s that can be merged by the code owner on his own responsibility. Merging a hotfix requires writing a desription on why it was necessary to merge it without review.
+2. When creating a PR, you should select **EXACTLY ONE** person as a reviewer. This way, a person selected as a reviewer knows he/she is the only one responsible for reviewing this PR. This person has to be someone who is working at the moment (not on vacation).
+If there is someone else who should see this PR, he/she should be assigned as Assignee on this PR. An Assignee is just an "interested party", someone who can review, but doesn't have to and has zero responsibility for a PR.
+	Reviewer Priority list:
+	1. Most senior developer on the project (excluding you).
+	2. Anyone else on the project.
+	3. Anyone else. (predetermined as a backup)
+3. After a reviewer is selected, responsibility for this PR is transfered to him/her. Reviewer has options to:
+	1. Do a Code Review and Approve PR.
+	2. Do a Code Review and Request Changes on the PR.
+	3. Return a response that he/she will be able to do Code Review some time later (say when).
+	4. Return a response that he/she wont be able to do a Code Review.
+4. After a reviewer has responded (3a,3b,3c,3d), the PR creator has to do one of the following:
```

Comment:
Suggestion - we should either completely remove inconsistent part `(3a,3b,3c,3d)` or change it to something like:
- `(3.1, 3.2, 3.3, 3.4)`
- "After a reviewer has responded _in any way mentioned in the previous step_, the PR creator..."

Reply:
@marojeee good suggestion, accepting  changes